### PR TITLE
Event type in model

### PR DESCRIPTION
**Pull request by karbmk**
Event type field added into models

### CODE REVIEW

File: openCurrents/models.py [hunk 1]
```diff
@@ -145,6 +145,17 @@ class Event(models.Model):
...
+    MANUAL = 'MN'
+    GROUP = 'GR'
+    OTHER = 'OT'
+    event_type_choices = (
+        (MANUAL, 'ManualTracking'),
+        (GROUP, 'Group'),
+        (OTHER, 'Other'),
+    )
+    event_type = models.CharField(max_length=2,
+                                      choices=event_type_choices,
```
Comment (nickolashe) [inline openCurrents/models.py]: indentation looks off

File: openCurrents/models.py [hunk 2]
```diff
@@ -145,6 +145,17 @@ class Event(models.Model):
     # coordinator contact info
     coordinator_firstname = models.CharField(max_length=128)
     coordinator_email = models.EmailField()
+    MANUAL = 'MN'
+    GROUP = 'GR'
+    OTHER = 'OT'
```
Comment (nickolashe) [inline openCurrents/models.py]: any thoughts what we would use this for?

File: openCurrents/views.py [hunk 3]
```diff
@@ -226,6 +226,7 @@ def track_hours(self, form_data):
         event = Event(
             project=self.project,
             description=form_data['description'],
+            event_type="MN",
```
Comment (nickolashe) [inline openCurrents/views.py]: is this the only place where we are creating events?

File: openCurrents/models.py (hunk 2 above)
Comment (karbmk) [inline openCurrents/models.py]: Yep this is how its filtered in the templates. They are basically the values of the tag.

File: openCurrents/models.py (hunk 1 above)
Comment (karbmk) [inline openCurrents/models.py]: Got it. Will make it proper.

File: openCurrents/views.py (hunk 3 above)
Comment (karbmk) [inline openCurrents/views.py]: The manual events. Yes. If they are being created elsewhere I have made GROUP the default. So it shouldn't be a problem.

File: openCurrents/models.py (hunk 2 above)
Comment (nickolashe) [inline openCurrents/models.py]: @karbmk i meant the 'OT' type specifically. i think lets leave it out until we have other types to deal with.

File: openCurrents/views.py (hunk 3 above)
Comment (nickolashe) [inline openCurrents/views.py]: got it, that will work

File: openCurrents/models.py (hunk 2 above)
Comment (karbmk) [inline openCurrents/models.py]: Oh sorry. I thought all of it. I will do the changes.

File: openCurrents/models.py [hunk 4]
```diff
@@ -154,8 +154,8 @@ class Event(models.Model):
         (OTHER, 'Other'),
     )
     event_type = models.CharField(max_length=2,
-                                      choices=event_type_choices,
-                                      default=GROUP)
+                                  choices=event_type_choices,
```
Comment (nickolashe) [inline openCurrents/models.py]: @karbmk we are using a slightly different practice for breaking lines and indenting, see this for an example:
https://github.com/opencurrents/opencurrents/blob/master/openCurrents/views.py#L312-L316

**Comment by nickolashe** [comment]
@karbmk you've tested this locally, correct?

**Comment by karbmk** [comment]
Yes

**Comment by nickolashe** [comment]
ok, feel free to merge one the indentation is consistent with the rest of the codebase

**Comment by karbmk** [comment]
Ok sure I will do that

**Pull request merged**